.travis.yml

```diff
@@ -18,35 +18,67 @@ php:
 
 env:
   matrix:
-    - TEST_SUITE=static-unit XDEBUG=true
-    - TEST_SUITE=integration
-    - TEST_SUITE=functional
-    - TEST_SUITE=functional FUNCTIONAL_INDEX=1
-    - TEST_SUITE=functional FUNCTIONAL_INDEX=2
-    - TEST_SUITE=functional FUNCTIONAL_INDEX=3
-    - TEST_SUITE=functional FUNCTIONAL_INDEX=4
-    - TEST_SUITE=functional FUNCTIONAL_INDEX=5
+    - TEST_SUITE=functional-ce
+    - TEST_SUITE=functional-ee
+    - TEST_SUITE=functional-ee FUNCTIONAL_INDEX=1
+    - TEST_SUITE=functional-ee FUNCTIONAL_INDEX=2
+    - TEST_SUITE=functional-ee FUNCTIONAL_INDEX=3
+    - TEST_SUITE=functional-ee FUNCTIONAL_INDEX=4
+    - TEST_SUITE=functional-ee FUNCTIONAL_INDEX=5
+
+stages:
+  - static-unit
+  - integration
+  - test
```

Comment:
Maybe `functional` rather than just `test`?

Reply:
Unless `test` is the shorthand to just run the `script:` section below?

Reply:
It's default Travis stage, haven't seen a good way to customize the name